pkg/auth/login.go

```diff
@@ -129,6 +129,7 @@ func filteredInformation(s []byte) []byte {
 	s = bytes.Replace(s, []byte("new-project"), []byte("project create"), -1)
 	s = bytes.Replace(s, []byte("<projectname>"), []byte("<project-name>"), -1)
 	s = bytes.Replace(s, []byte("project <project-name>"), []byte("project set <project-name>"), -1)
+	s = bytes.Replace(s, []byte("projects"), []byte("project list"), -1)
```

Comment:
I would suggest use "\"odo project list\"" so that we specifically check `"odo project list"` to be very specific

Reply:
what I saying is to consider having `"` (quotes) so that unexpected changes dont show as projects is a very common word used at multiple places

Reply:
Using 
```
s = bytes.Replace(s, []byte("\"odo projects\""), []byte("\"odo project list\""), -1)
```

doesn't replace the string.